templates/lxd/install.html

```diff
@@ -0,0 +1,232 @@
+          {{ image (
+            url="https://assets.ubuntu.com/v1/9c80823b-confinement.svg",
+            alt="",
+            width="263",
+            height="150",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="p-section--shallow u-fixed-width">
+    <hr class="p-rule">
+    <h2>Familiarise yourself with&nbsp;the&nbsp;basics</h2>
+  </div>
+  <div class="row--25-75">
+    <div class="col">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Instances</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <p>LXD supports two types of instances: system containers and virtual machines.</p>
+          <p>
+            <a href="https://documentation.ubuntu.com/lxd/en/latest/explanation/containers_and_vms/">Learn more about containers and VMs&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <hr class="p-rule col-9 col-medium-6">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Security</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <p>Security is at the forefront of everything we do and there are various things to consider to keep your LXD installation secure.</p>
+          <p>
+            <a href="https://documentation.ubuntu.com/lxd/en/latest/explanation/containers_and_vms/">Learn more about LXD and security&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <hr class="p-rule col-9 col-medium-6">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Images</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <p>LXD uses an image-based workflow, providing images for a large number of Linux distributions.</p>
+          <p>
+            <a href="https://documentation.ubuntu.com/lxd/en/latest/image-handling/">Learn more about common operations with images&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <hr class="p-rule col-9 col-medium-6">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Contribute</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <p>We welcome and value contributions from the community</p>
+          <div class="p-strip--shallow">
+            <p>
+              <a href="https://documentation.ubuntu.com/lxd/en/latest/contributing">How to contribute to LXD&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--white is-deep">
+  <div class="row">
+    <div class="col-9 col-medium-4 col-start-large-4 col-start-medium-3">
+      <h2><a href="/contact-us" aria-controls="lxd-install-modal">Talk to our LXD experts&nbsp;&rsaquo;</a></h2>
```

Comment:
This links to a root level contract us page. It's that correct?

Reply:
That's what we do on all modal forms on c.com. The link triggers a modal already, so I don't see the need to create a separate form that looks the same as the modal form.